sql-scripts/US004-view-band-level.sql

```diff
@@ -0,0 +1,32 @@
+USE IWantItDataWay_StephenH;
+
+CREATE TABLE `band` (
+	band_id tinyint unsigned,
+	band_name VARCHAR(50) NOT NULL,
+	PRIMARY KEY (band_id)
+);
+
+INSERT INTO band (band_id, band_name) VALUES (1,"Apprentice"), (2,"Trainee"), (3,"Associate"), (4,"Senior Associate"), (5,"Consultant"), (6,"Manager"), (7,"Principal"), (8,"Leadership Community");
```

Comment:
I can't remember what we discussed the other day, but the band numbers go the opposite direction

Reply:
In that case it might be best to change the ID back to an auto increment value (I know I asked for it not to be that before 😅)